crypter.go

```diff
@@ -532,7 +532,8 @@ func (obj JSONWebEncryption) DecryptMulti(decryptionKey interface{}) (int, Heade
 
 	// The "zip" header parameter may only be present in the protected header.
 	if comp := obj.protected.getCompression(); comp != "" {
-		plaintext, _ = decompress(comp, plaintext)
+		plaintext, err = decompress(comp, plaintext)
+		return -1, Header{}, nil, fmt.Errorf("go-jose/go-jose: failed to decompress plaintext: %w", err)
```

Comment:
Shouldn't this be
```suggestion
		if err != nil {
			return -1, Header{}, nil, fmt.Errorf("go-jose/go-jose: failed to decompress plaintext: %w", err)
		}
```

Reply:
Ack, yes. I had fixed this in my latest branch, along with adding a test. But I pushed to the wrong place 🤦🏻